frontend/src/main/components/RequestType/RequestTypeForm.js

```diff
@@ -35,7 +35,7 @@ function RequestTypeForm({
         </Form.Group>
       )}
 
-      <Form.Group className="mb-3">
+      {/* <Form.Group className="mb-3">
```

Comment:
What is "commented out code" doing in a PR that's marked "Ready For Staff"?

And why did both the original PR author and the person that did the code review think this was ok?

It's literally the first thing in the list of PR Criteria.

https://ucsb-cs156.github.io/topics/pull_requests/#pr-checklist

It's kind of astonishing to still see that at this stage.